girder/models/model_base.py

```diff
@@ -274,7 +274,12 @@ def remove(self, document, **kwargs):
 
         event = events.trigger('.'.join(('model', self.name, 'remove')),
                                document)
-        if not event.defaultPrevented:
+        kwargsEvent = events.trigger(
+            '.'.join(('model', self.name, 'remove_with_kwargs')), {
+                'document': document,
+                'kwargs': kwargs
+            })
+        if not event.defaultPrevented and not kwargsEvent.defaultPrevented:
```

Comment:
It's more a question of the behavior of stopPropagation, but it is my sincere hope that nobody ever calls that on a model removal event since I can't imagine that would ever be a good idea.